docs/sources/explore/_index.md

```diff
@@ -83,6 +83,10 @@ You can then click on any panel icon in the content outline to navigate to that
 
 When using Explore, the URL in the browser address bar updates as you make changes to the queries. You can share or bookmark this URL.
 
+{{% admonition type="note" %}}
+Explore may generate relatively long URLs, some tools, like messaging or videoconferencing apps, may truncate messages to a fixed length. In such cases Explore will display a warning message and load a default state. If you encounter issues when sharing Explore links in such apps, you can generate shortened links. See [Share shortened link]("#share-shortened-link"}}) for more information.
```

Comment:
This link doesn't work. Generates a URL to http://localhost:3002/docs/grafana/latest/explore/%22#share-shortened-link%22%7d%7d

Reply:
should be fixed now, mind double checking?